src/main/java/services/AuthService.java

```diff
@@ -71,7 +69,7 @@
         }
 
         if (!error) {
-            //Check on unique values in DB
+            //Check on unique values in DB.
             if (dao.isExist("login", user.getLoginName())) {
```

Comment:
Do you use only one DAO for all entities? It'll be better to split it up

Reply:
Ok, we will split.
About checking unique values, is this ok that we send in method with 2 strings, one is colum in table, another value. Or better create each method for each columns?

Reply:
it'll be better to create separate methods